vector/src/main/res/layout/view_voice_message_recorder.xml

```diff
@@ -231,10 +233,11 @@
         android:layout_height="wrap_content"
         android:layout_marginBottom="84dp"
         android:visibility="gone"
+        android:accessibilityLiveRegion="polite"
```

Comment:
This could become a bit annoying to speak out the text each time the view become visible, since the "toast" is displayed each time a voice recording starts, and also in some other circumstances. Maybe we should improve that in the code.

Reply:
I haven't strong objection regarding this. My thinking is that this way toast gets spoken whenever it is shown and hint on how to operate this feature will be easier to discover for screen reader users. Other idea is to implement custon AccessibilityNodeInfo class for the mic button, adding the double tap accessibility action with appropriate description. The latter is more difficult to implement so I went with this initially. If you like I can revert this part and we can decide  later.

Reply:
Ok, I see your point, thanks for the fast feedback. Let's keep it like that for now.